Components/Image/image.css

```diff
@@ -0,0 +1,10 @@
+
+.responsive-image {
+    min-width: 100%;
+    height: auto;
+}
+.image-rd {
```

Comment:
Give the class a more descriptive name like `.image-round`. Much easier to read and understand.

Reply:
okay, Thanks